plugins/commands/requirements/src/main/kotlin/RequirementsCommand.kt

```diff
@@ -80,17 +71,10 @@ class RequirementsCommand(
         UNAVAILABLE
     }
 
-    private val list by option(
-        "--list", "-l",
-        help = "A comma-separated list of requirements to list."
-    ).enum<RequirementsType>().split(",").default(RequirementsType.entries)
-
     private val reflections by lazy { Reflections("org.ossreviewtoolkit", Scanners.SubTypes) }
 
     override fun run() {
-        if (RequirementsType.PLUGINS in list) listPlugins()
-
-        val status = if (RequirementsType.COMMANDS in list) checkToolVersions() else enumSetOf(VersionStatus.SATISFIED)
+        val status = checkToolVersions()
```

Comment:
Commit message, maybe add https://github.com/oss-review-toolkit/ort/pull/9140#issuecomment-2656467942 as a reference?

Reply:
This commit should also update the following locations:

```
.github/ISSUE_TEMPLATE/bug.md:Output of the `ort requirements -l commands` command:
cli/src/main/dist/plugin/README.md:Afterward, you can run `ort requirements --list plugins`
integrations/jenkins/Jenkinsfile:                /opt/ort/bin/ort $ORT_OPTIONS requirements --list plugins
```

Reply:
And the shell completion scripts need to be updated.